### ISSUE TITLE
Failed to load the folded-code-preview package

### ISSUE THREAD

**Issue by mohalobaidi** [73CN0109y-Packages/folded-code-preview#1]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [folded-code-preview](https://github.com/73CN0109y-Packages/folded-code-preview) package 0.2.2

### Stack Trace

Failed to load the folded-code-preview package

```
At Cannot find module 'jquery'

Error: Cannot find module 'jquery'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:354:58)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/folded-code-preview/lib/folded-code-preview.js:4:11)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (Applications/Atom.app/Contents/Resources/app/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:96:26)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/src/package.js:887:35)
    at /Applications/Atom.app/Contents/Resources/app/src/package.js:158:34
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:99:21)
    at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app/src/package.js:144:18)
    at PackageManager.module.exports.PackageManager.loadAvailablePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:565:22)
    at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:495:27)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:643:36)
    at /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:519:35
    at exit (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:128:22)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:338:17)
    at /Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:351:17
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:220:15)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

### Commands

```
  2x -0:52.8.0 core:backspace (input.hidden-input)
     -0:51.5.0 editor:newline (input.hidden-input)
     -0:51 core:backspace (input.hidden-input)
     -0:48.3.0 editor:newline (input.hidden-input)
     -0:48 snippets:next-tab-stop (input.hidden-input)
     -0:48 snippets:expand (input.hidden-input)
     -0:48 editor:indent (input.hidden-input)
     -0:45.5.0 core:move-right (input.hidden-input)
     -0:45.3.0 editor:newline (input.hidden-input)
     -0:42.6.0 core:save (input.hidden-input)
     -0:38 core:backspace (input.hidden-input)
     -0:35.5.0 grammar-selector:show (atom-text-editor.editor.is-focused)
  5x -0:34.4.0 core:backspace (input.hidden-input)
     -0:31.9.0 core:confirm (input.hidden-input)
     -0:17.5.0 core:paste (input.hidden-input)
     -0:16.6.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.3 
atom-clock 0.1.12 
atom-css-comb 3.2.0 
atom-pug 0.2.0 
emmet 2.4.3 
file-icons 2.1.8 
folded-code-preview 0.2.2 
platformio-ide-terminal 2.5.3 
```

**Comment by Tecnology73** [comment]
This should be solved in the latest version :) Let me know if it continues.

**Comment by Gabb1995** [comment]
Just tried installing today, happened the second I installed the package, running on latest Atom version on linux.

Happens everytime I restart atom too.

Steps to reproduce:

1. Enable plugin
2. Restart atom

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 17.04
**Thrown From**: [folded-code-preview](https://github.com/73CN0109y-Packages/folded-code-preview) package 0.2.3


### Stack Trace

Failed to load the folded-code-preview package

```
At Cannot find module 'jquery'

Error: Cannot find module 'jquery'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/usr/share/atom/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/usr/share/atom/resources/app/src/module-cache.js:354:58)
    at Module.require (usr/share/atom/resources/app/static/index.js:40:43)
    at require (/usr/share/atom/resources/app/src/native-compile-cache.js:66:33)
    at /packages/folded-code-preview/lib/folded-code-preview.js:4:11)
    at Module.get_Module._compile (/usr/share/atom/resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/usr/share/atom/resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (usr/share/atom/resources/app/static/index.js:47:45)
    at require (internal/module.js:20:19)
    at customRequire (/usr/share/atom/resources/app/static/<embedded>:96:26)
    at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app/src/package.js:887:35)
    at /usr/share/atom/resources/app/src/package.js:158:34
    at Package.module.exports.Package.measure (/usr/share/atom/resources/app/src/package.js:99:21)
    at Package.module.exports.Package.load (/usr/share/atom/resources/app/src/package.js:144:18)
    at PackageManager.module.exports.PackageManager.loadAvailablePackage (/usr/share/atom/resources/app/src/package-manager.js:565:22)
    at /usr/share/atom/resources/app/src/package-manager.js:479:25
    at Config.module.exports.Config.transact (/usr/share/atom/resources/app/src/config.js:321:22)
    at PackageManager.module.exports.PackageManager.loadPackages (/usr/share/atom/resources/app/src/package-manager.js:473:25)
    at /usr/share/atom/resources/app/src/atom-environment.js:812:34
```

### Commands

```
  2x -0:09.5.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
aligner 1.2.3 
aligner-css 1.2.1 
aligner-javascript 1.2.0 
aligner-php 1.1.1 
atom-autocomplete-php 0.25.6 
atom-symfony2 0.3.0 
atom-sync 0.7.3 
busy-signal 1.4.3 
docblockr 0.10.5 
file-icons 2.1.8 
folded-code-preview 0.2.3 
git-plus 7.9.3 
intentions 1.1.2 
linter 2.2.0 
linter-phpcs 1.6.7 
linter-ui-default 1.6.2 
minimap 4.28.3 
minimap-autohide 0.10.1 
minimap-git-diff 4.3.1 
minimap-linter 2.0.0 
minimap-selection 4.4.0 
php-twig 4.0.0 
pigments 0.39.1 
sync-settings 0.8.2 
```

**Comment by cliffordfajardo** [comment]
I solved the jQuery issue `Cannot find module 'jquery'` by running `npm i -S jQuery` in `folded-code-preview`'s folder.
Next, I got another error message about another missing dependency  `Cannot find module 'etch'*` so I ran `npm i -S etch`

I reloaded Atom and then the package started working again.

**Comment by Tecnology73** [comment]
This should really be solved now in the latest version (0.2.4)
